### PR TITLE
PSTRESS-137 Add support for ALTER INSTANCE RELOAD KEYRING DDL in pstress

### PR DESCRIPTION
**Pull request by mohitj1988**

### CODE REVIEW

**Review by rahulmalik87 — CHANGES_REQUESTED** [review]

File: src/help.cpp [hunk 1]
```diff
@@ -361,6 +361,13 @@ void add_options() {
   opt->setSQL();
   opt->setDDL();
 
+  /* Reload keyring component configuration */
+  opt = newOption(Option::INT, Option::ALTER_INSTANCE_RELOAD_KEYRING, "reload-keyring");
+  opt->help = "Alter instance reload keyring";
+  opt->setInt(0);
```
[the remark this answers is not in the document]
Comment (mohitj1988) [reply src/help.cpp:367]: We don't want to enable this DDL by default. The purpose of this DDL comes into affect when a user changes the global or local manifest file and wants to reload the new configuration to be read from manifest files. Enabling this DDL will not achieve any purpose unless we are testing keyring component related feature.

[the remark this answers is not in the document]
Comment (rahulmalik87) [reply src/help.cpp:367]: Even if the manifest file doesn't change, Reloading will execute some code path which can tested with another concurrent load.
Like how it behaves with ALTER INSTANCE ROTATE KEY. 
 For this reason, I would enable it by default with a very low probability.

[the remark this answers is not in the document]
Comment (rahulmalik87) [reply src/help.cpp:367]: @mohitj1988 Please address this whenever you get time.

[the remark this answers is not in the document]
Comment (mohitj1988) [reply src/help.cpp]: Fixed. Kindly check

**Review by rahulmalik87 — COMMENTED** [review]
Pstress disables encryption-related options if --no-encryption is passed. can also disable "Alter instance reload kyering"  if --no-encryption is given.

**Review by rahulmalik87 — APPROVED** [review]
LGTM

**Pull request merged**